server/src/main/java/org/elasticsearch/transport/TransportLogger.java

```diff
@@ -77,16 +77,22 @@ private static String format(TcpChannel channel, BytesReference message, String
                 final byte status = streamInput.readByte();
                 final boolean isRequest = TransportStatus.isRequest(status);
                 final String type = isRequest ? "request" : "response";
-                final String version = Version.fromId(streamInput.readInt()).toString();
+                Version version = Version.fromId(streamInput.readInt());
+                final String versionString = version.toString();
```

Comment:
just drop this variable and use `sb.append(", version: ").append(version)` further down below